components/textbook-beta/DynamicBackgroundLogo.vue

```diff
@@ -0,0 +1,251 @@
+<template>
+  <div ref="canvasWrapper" class="dynamic-background-logo">
+    <canvas ref="canvas" class="dynamic-background-logo-background-canvas" />
+    <SketchedLogo class="dynamic-background-logo-overlay" />
```

Comment:
Small change follow our BEM convention:

```suggestion
    <SketchedLogo class="dynamic-background-logo__overlay" />
```